### DISCUSSION TITLE
How to segregate error types in brew?

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [Homebrew/discussions#75] [Everyday usage] [answered]
Hello! I want to segregate error types in brew while installing new formulas:
- no internet connection
- smth is already installed

The problem root is that brew returns 1 in both cases. Perhaps I can use grep as workaround but I don't consider it as a good decision.

**Comment by SMillerDev** [comment, marked as answer]
You can check what's installed before you run the install by using `brew ls formula`
